bin/shutter

```diff
@@ -4502,7 +4505,13 @@ sub STARTUP {
 		#main
 		$settings{'general'}->{'filetype'} = $combobox_type->get_active;
 		$settings{'general'}->{'quality'}  = $scale->get_value();
-		$settings{'general'}->{'filename'} = $filename->get_text();
+		if ($filename->get_text() =~ "\%NN" || $filename->get_text() =~ "\%T") {
+			$settings{'general'}->{'filename'} = $filename->get_text();
+		} else {
+			$sd->dlg_error_message($@, $d->get("Settings could not be saved! Please make sure that the filename contains a wildcard like \%NN or \%T!"));
+			evt_show_settings();
+			$settings_sane = 0;
```

Comment:
could do an early return instead?

Reply:
You mean, instead of the whole $settings_sane check? I think, this might work, I will try that!